### ISSUE TITLE
Trained Faster-RCNN model with 200 classes of LSVRC detection competition - caffe

### ISSUE THREAD

**Issue by rockNroll87q** [ShaoqingRen/faster_rcnn#74]
Hello to everyone!

For my research purposes, I'm looking for a faster-RCNN model, already trained with 200 classes of LSVRC detection competition.
Could someone suggest me where to find an already-trained model, or even better share it with me?

Thanks!

**Comment by lerasht** [comment]
i'm also loolking for this, did you find?

**Comment by rockNroll87q** [comment]
Hi. Not yet. As you can see none reply.

**Comment by xiaoxiongli** [comment]
why not using the 1000 classes of LSVRC model?

**Comment by rockNroll87q** [comment]
@xiaoxiongli There is a _detection_ model trained on 1000 classes?

**Comment by xiaoxiongli** [comment]
@rockNroll87q  yes, there are so many imagenet(1000 category) pre-trained model. please google "imagenet" or "caffe model zoo"
